tests/integration/test_reads.py

```diff
@@ -873,3 +873,48 @@ def test_table_scan_empty_table(catalog: Catalog) -> None:
     result_table = tbl.scan().to_arrow()
 
     assert len(result_table) == 0
+
+
+@pytest.mark.integration
+def test_plan_tasks(session_catalog: Catalog) -> None:
```

Comment:
nit: should we also test `size_in_bytes` and the relationship between `READ_SPLIT_SIZE`, `READ_SPLIT_LOOKBACK`, and `READ_SPLIT_OPEN_FILE_COST`?